### ISSUE TITLE
My Addon

### ISSUE THREAD

**Issue by xoptimus** [v137/kodi#3]
Hi I have Notice You Have Removed My Name From Demo UHD That Addon Was Created by me pipcan and as you have made no changes i dont see why you have removed my name and added yours i would like you to put my name back

**Comment by v137** [comment]
Sorry, was just testing, have removed the addon from my repo.

**Comment by v137** [comment]
By the way, the version I downloaded was from SuperRepo. It only lists Oneadvent as author. If there is some issue please contact SuperRepo admins.

**Comment by xoptimus** [comment]
look i an saying dont use it or even edit it but credits were credits due

**Comment by v137** [comment]
I agree with you. The version I forked came from SR and they did not credit accordingly it seems.
